space_commands/Readme.txt

```diff
@@ -0,0 +1,7 @@
+Space_launches command uses an api with limited allowed calls per time.
+To run tests while developing: run this simple http server on your own machine when testing space_launches command. Set the URL to the test URL: Comment out the production one.
```

Comment:
Is this still necessary now that the feature is finished?

Reply:
Not necessary, so it's included in the .ignore so it won't show up in the commit anymore :-)